Library/Formula/negfix8.rb

```diff
@@ -0,0 +1,19 @@
+class Negfix8 < Formula
+  desc "Negfix8 turns scanned negative images into positives."
+  homepage "https://sites.google.com/site/negfix"
+  url "https://sites.google.com/site/negfix/downloads/negfix8.3.tgz"
+  sha256 "2f360b0dd16ca986fbaebf5873ee55044cae591546b573bb17797cbf569515bd"
+
+  depends_on "imagemagick" => %w(with-libtiff with-quantum-depth-16)
```

Comment:
`with-libtiff` is the default now (as of today) so it should be removed here. You can just use `depends_on "imagemagick" => "with-quantum-depth-16"`.